package.json

```diff
@@ -34,7 +34,7 @@
     "dev": "start-storybook -p 6006",
     "build": "rm -fr ./dist ./native && rollup -c && npm run test:size",
     "deploy:docs": "storybook-to-ghpages",
-    "test": "npm run test:tsc && npm run test:unit && npm run build",
+    "test": "npm run test:tsc && npm run test:unit",
```

Comment:
It's understandable you've removed this script, once the build step is not part of the test script. But to test the bundle size, it still needs to run the build script to measure the final bundle files. So in order to be more clear the "bundlesize" step, I'd recommend the following changes:
- Build script: removes the `test:size` from there;
- Create a pre-test script: which will run the build one;
- Test script: adds `test:size`.

Does it make sense? Let me know your thought.

Reply:
I guess it will require to add the bundlesize env in the test script.

Reply:
After it runs the testing step, it runs a build step, where it would trigger the `test:size` script. It's just removed from the test step
https://github.com/danilowoz/react-content-loader/pull/217/files#diff-e9f950f17198d3d5e3122a44230a09b9R46-R50